modules/ROOT/pages/amazon/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,25 @@
+= Amazon SQS Additional Configuration Information - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+
+== Creating the FIFO Queue
+
+To create the FIFO queue, use the *Create queue* operation in the connector, and set the *FifoQueue* and *ContentBasedDeduplication* field values to true.
+
+End the *queue name* with the `.fifo` extension. For example,  `MyTestFIFOQueue.fifo`.
+
+While creating the FIFO queues, choose either of the two regions: US East (Ohio) or US West (Oregon). Amazon only supports these two regions for FIFO queues.
+
+When using Amazon SQS Connector to send messages to the FIFO queue, use the same queue URL and region name that you used when creating the FIFO Queue. You also have to provided a value for the attribute `message group id` in the connector configuration while sending messages.
```

Comment:
You must also provide a value for the `message group id` attribute in the connector configuration.